examples/vaadin-lts-v10-simple/pom.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I've tried to compile the vaadin-lts-v10-simple/ and vaadin-vxx-simple/ examples and I get this error on both:

> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-war-plugin:3.1.0:war (default-war) on project vaadin-lts-v10-simple: Error assembling WAR: webxml attribute is required (or pre-existing WEB-INF/web.xml if executing in update mode) -> [Help 1]